### DISCUSSION TITLE
When I try to open the DLL file, it becomes unresponsive.

### DISCUSSION THREAD

**Discussion by izkluxcvy** [rizinorg/cutter#2972] [Q&A]
When I open `FLEngine_x64.dll` , which is bundled with the software FL Studio 20, without analysis, it stops responding after about 1 second of loading.
The environment is windows 10 and the cutter was the same whether it was zip or choco.
I think it is supported, but it does not work.
Is there some other setting that needs to be done? Thank you in advance.

**Comment by ret2libc** [comment]
Hi!
Can you share the binary? How big is it? Can you provide a crash dump, if there's any?
